hi-ml-histopathology/src/histopathology/configs/classification/DeepSMILEPanda.py

```diff
@@ -36,26 +31,39 @@
     LoadTilesBatchd,
 )
 
+from histopathology.configs.classification.BaseMIL import BaseMIL
+from histopathology.datasets.panda_dataset import PandaDataset
+from histopathology.models.deepmil import DeepMILModule
+
 
 class DeepSMILEPanda(BaseMIL):
+    """`is_finetune` sets the fine-tuning mode. If this is set, setting cache_mode=CacheMode.NONE takes ~30 min/epoch and
+    cache_mode=CacheMode.MEMORY, precache_location=CacheLocation.CPU takes ~[5-10] min/epoch.
+    Fine-tuning with caching completes using batch_size=4, max_bag_size=1000, num_epochs=20, max_num_gpus=1 on PANDA.
+    """
     def __init__(self, **kwargs: Any) -> None:
         default_kwargs = dict(
             # declared in BaseMIL:
             pooling_type=GatedAttentionLayer.__name__,
             # average number of tiles is 56 for PANDA
             encoding_chunk_size=60,
+            cache_mode=CacheMode.MEMORY,
+            precache_location=CacheLocation.CPU,
+            is_finetune=False,
 
             # declared in DatasetParams:
-            local_datasets=[Path("/tmp/datasets/PANDA_tiles"), Path("/tmp/datasets/PANDA")],
-            azure_datasets=["PANDA_tiles", "PANDA"],
+            local_dataset=Path("/tmp/datasets/PANDA_tiles"),
```

Comment:
These 4 variable names will also need to be changed (see how the file was previously - azure_dataset_id and extra_azure_dataset_ids can now be provided as 1 list called 'azure_datasets' and similarly, local_dataset and extra_local_dataset_paths can be provided as 1 list called 'local_datasets', and num_epochs is now called max_epochs)